project.clj

```diff
@@ -13,6 +13,7 @@
   {:dev
    {:aliases {"lint" ["do" ["whitespace-linter"] ["eastwood"]]}
     :dependencies [[com.gfredericks/test.chuck "0.2.8"]
+                   [eftest "0.3.1"]
```

Comment:
A more effective way of doing this might be to run the cache step after we've run our tests.